.buildkite/longruns/pipeline.yml

```
@@ -82,6 +98,14 @@ steps:
           CLIMACORE_DISTRIBUTED: "MPI"
         agents:
           slurm_ntasks: 64
+      
+      - label: ":computer: held suarez (ρe_tot) equilmoist high resolution (dcmip topography)"
```

```suggestion
      - label: ":computer: held suarez (ρe_tot) equilmoist high resolution topography (dcmip)"
```
(Just making sure they are consistent with each other)